modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -859,29 +916,27 @@ def dump_ntds_hashes
     ph_drs = dcerpc_client.drs_bind
     dc_infos = dcerpc_client.drs_domain_controller_info(ph_drs, domain_name)
     user_info = {}
-    dc_infos.each do |dc_info|
-      users.each do |user|
-        sid = user[0]
-        crack_names = dcerpc_client.drs_crack_names(ph_drs, rp_names: [sid])
-        crack_names.each do |crack_name|
-          user_record = dcerpc_client.drs_get_nc_changes(
-            ph_drs,
-            nc_guid: crack_name.p_name.to_s.encode('utf-8'),
-            dsa_object_guid: dc_info.ntds_dsa_object_guid
-          )
-          user_info[sid] = parse_user_record(dcerpc_client, user_record)
-        end
+    dc_info = dc_infos[0]
```

Comment:
Not sure - as far as I understand the API, if there's any effect it would likely be on detection rather than functionality; but not sure whether changing it would make it more or less likely to be detected. Don't think it's worth changing without evidence one way or the other.